analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -81,6 +81,14 @@
     from the source content are removed.
   * 'column numbers' from the main diag sections location.
 
+- diagnostic-message:
```

Comment:
Hmm, perhaps we should highlight that this is the same as `context-free-v2` + bug path messeges?
```
- diagnostic-message:
  * context-free-v2
  * 'bug step messages' from all events.
```